### DISCUSSION TITLE
Importing different repo's through yarn install (SSH key issues?)

### DISCUSSION THREAD

**Discussion by DTufto** [codesandbox/codesandbox-client#6824] [Q&A]
Hello! Today I started porting over a repository to codesandbox, and I ran into the following problem:
Yarn install cannot run because of SSH key problems.

The situation is as follows:
We have a "base" frontend repo, where we create products which are not specific to any clients.
We have clientspecific repo's, where we add code that is only to be used for that client.

The clientspecific repo imports the "base" frontend with yarn install, but codesandbox cannot access it.
I thought of the usual method on Ubuntu by creating an ssh-key in the Codesandbox-terminal and adding it to my github account. (Which works, as you can see in the following screenshot.)
![image](https://user-images.githubusercontent.com/58273766/184195878-84f51093-5be4-49f2-b1ee-d49cb1418f25.png)

But I keep getting the following error message.
![image](https://user-images.githubusercontent.com/58273766/184196892-b6e86a48-dfff-40e0-b640-2ca679786f7e.png)

Could anyone help me out? Thanks!

**Comment by JamesACS** [comment]
Hey @DTufto - Have you uploaded the custom repo to NPM? You should be able to then add it. 

If the repo is private, you'll also need to upgrade to Team Pro.
